plugins/PrefixDiscussion/class.prefixdiscussion.plugin.php

```diff
@@ -186,6 +186,11 @@ public function discussionController_beforeDiscussionRender_handler($sender) {
         if ($prefix == '') {
             return;
         }
+        
+        // Purify the prefix.
+        $formatter = Gdn::factory($formatMethod.'Formatter')) {
```

Comment:
I think the proper way to handle formatting nowadays is `Gdn_Format::to()` the factory was never updated to support Rich Editor.